parser/include/parser/abstractparser.h

```diff
@@ -44,6 +44,16 @@ class AbstractParser
    * @return Returns true if the parse succeeded, false otherwise.
    */
   virtual bool parse() = 0;
+  /**
+   * Returns true in case database indices are required for the parser, due to performance reasons.
+   *
+   * Should return the same value on each call for the same object.
+   * @return
```

Comment:
Please write something in this line or delete it.

Reply:
Fixed.